R/utils.R

```diff
@@ -5,7 +5,8 @@
 #' @keywords info
 
 available_metrics <- function() {
-  return(unique(scoringutils::metrics$Name))
+  return(unique(c(scoringutils::metrics$Name,
```

Comment:
I think the right place to break this line is after return( and then unique( and so on vs randomly at the end.